argostranslate/translate.py

```diff
@@ -73,7 +73,7 @@ def split_into_paragraphs(input_text):
         """
         return input_text.split('\n')
 
-    def combine_paragraphs(self, paragraphs):
```

Comment:
This was intentional. I'm trying to reduce inheritance in the codebase and made this static. There are probably ways to improve this though, maybe a "static" method decorator?

Reply:
Yes, indeed,
That's why I change this line to make it static (`self` shouldn't be a parameter for static classes).
Yep, we can add `@staticmethod`, that makes the method callable from the object too, as well as the current way that we are calling it.

Reply:
Just realized I read this wrong, you're right. The `@staticmethod` decorator is a nice addition though.